docs/standard/microservices-architecture/architect-microservice-container-applications/scalable-available-multi-container-microservice-applications.md

```diff
@@ -64,17 +64,17 @@ Azure Service Fabric
 
 ![https://azure.microsoft.com/svghandler/service-fabric?width=600&height=315](./media/image26.png)
 
-> [Service Fabric](https://azure.microsoft.com/en-us/documentation/articles/service-fabric-overview/) is a Microsoft microservices platform for building applications. It is an [orchestrator](https://azure.microsoft.com/en-us/documentation/articles/service-fabric-cluster-resource-manager-introduction/) of services and creates clusters of machines. By default, Service Fabric deploys and activates services as processes, but Service Fabric can deploy services in Docker container images. More importantly, you can mix services in processes with services in containers in the same application.
+> [Service Fabric](https://azure.microsoft.com/documentation/articles/service-fabric-overview/) is a Microsoft microservices platform for building applications. It is an [orchestrator](https://azure.microsoft.com/documentation/articles/service-fabric-cluster-resource-manager-introduction/) of services and creates clusters of machines. By default, Service Fabric deploys and activates services as processes, but Service Fabric can deploy services in Docker container images. More importantly, you can mix services in processes with services in containers in the same application.
```

Comment:
- change https://azure.microsoft.com/documentation/articles/service-fabric-overview/ to https://docs.microsoft.com/azure/service-fabric/service-fabric-overview
-  change https://azure.microsoft.com/documentation/articles/service-fabric-cluster-resource-manager-introduction/ to https://docs.microsoft.com/azure/service-fabric/service-fabric-cluster-resource-manager-introduction